ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,209 @@
+                        </msItem>
+                        <msItem xml:id="ms_i1.6">
+                            <locus from="128v" to="140r"/>
+                            <title type="complete"><foreign xml:lang="gez">Śalast antiphons</foreign></title>
+                        </msItem>
+                        <msItem xml:id="ms_i1.7">
+                            <locus from="140r" to="148v"/>
+                            <title type="complete"><foreign xml:lang="gez"> ʾAryām antiphons</foreign></title>
+                        </msItem>
+</msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="149r" to="173v"/>
+                            <title type="complete" ref="LIT2610Zemmar">Zǝmmāre 
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="175r" to="195v"/>
+                            <title type="complete" ref="LIT3186Meeraf">Mǝʿrāf, Common of the Office 
+                                
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="195v" to="209r"/>
+                            <title type="complete" ref="LIT1990Mawase">Mawāśǝʾt, Antiphonary for feasts and funeral service
+                            </title>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="209">209</measure>
+                                    <dimensions type="outer"/>
+                                    <height unit="mm">375</height>
+                                    <width unit="mm">320</width>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="46">   
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc><persName
+                                    ref="PRS1071dAbbadi"/>describes the handwriting as ʻneat and niceʼ<foreign xml:lang="fr">(ʻfine et jolieʼ)</foreign>.    
```

Comment:
space is missing between the name and the text and between the translation and the original (as it is now, it would be printed
`D'Abbadiedescribes the handwriting as 'neat and nice'('fine et jolie')`)

Reply:
Ok! Thank you!